blog-meeting-minutes/2024-02-09-office-hour.md

```diff
@@ -0,0 +1,36 @@
+---
+slug: office-hour-09-02-2024
+title: Office Hour 09.02.2024
+authors: 
+    - tomasz_barwicki
```

Comment:
I think you also need to update the [authors.yaml](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/blog-meeting-minutes/authors.yaml)

Reply:
thanks, done! https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/664/commits/874ca12df53cda154b993d24fc92523462934ae4